build/android/src/main/res/layout/assetcopy.xml

```diff
@@ -18,7 +18,7 @@
         android:ellipsize="middle" 
         android:singleLine="true"
         android:layout_gravity="center_horizontal"
-        android:text="preparing media ..."
+        android:text="@string/preparing_media"
```

Comment:
How long is this text shown? A milisecond? Ten? Do we really need to translate it?